modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -10,7 +10,7 @@
 
 [NOTE]
 ====
-This Operator is only fully supported for Microsoft Azure Stack Hub, {rh-openstack-first}, and VMware vSphere.
+This Operator is fully supported for Microsoft Azure Stack Hub, Nutanix, {rh-openstack-first}, and VMware vSphere.
```

Comment:
when we say fully supported, are we talking about the GA release status?

Reply:
Good point, better to just say GA when we mean GA. Thanks!